app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -87,7 +89,7 @@ import { addSearchSuggestion } from "discourse/widgets/search-menu-results";
 import { CUSTOM_USER_SEARCH_OPTIONS } from "select-kit/components/user-chooser";
 
 // If you add any methods to the API ensure you bump up this number
-const PLUGIN_API_VERSION = "0.12.3";
+const PLUGIN_API_VERSION = "0.13.0";
```

Comment:
I'm pretty sure we follow semver at Discourse so any new functionality should be bumping the minor verison.